### ISSUE TITLE
Toast pop callback usage is weird

### ISSUE THREAD

**Issue by CatchABus** [zerodevx/svelte-toast#31]
```
// Remove all toasts from "new" container
toast.pop((i) => i.target !== 'new')
```
I believe this functionality is contradicting itself.
Callback should delete toasts that meet the condition instead.
At first look, if comment wasn't there, one would parse this line of code as `Pop all toasts whose target is NOT new`.

The following approach would be more understandable.
```
// Remove all toasts from "new" container
toast.pop((i) => i.target === 'new')
```

**Comment by zerodevx** [comment]
Hey, thanks for the feedback - I agree that it's misleading. The API should be cleaned up by the next major release so it's more intuitive. Internally, when the `pop()` method detects that the input is a filter function, it simply runs it against the store. So:

```js
toast.pop(i => i.target !== 'new')
```

is actually equivalent to:

```js
import { toast } from '@zerodevx/svelte-toast'

$toast = $toast.filter(i => i.target !== 'new')
```

Hope this gives some context.

**Comment by zerodevx** [comment]
Fixed by #66 with a [clearer way](https://github.com/zerodevx/svelte-toast#removing-multiple-toasts) to clear toasts from a particular container.
